modules/InboundEmail/InboundEmail.php

```diff
@@ -4407,7 +4407,7 @@ public function importDupeCheck($message_id, $header, $textHeader)
         $r = $this->db->query($query, true);
         $a = $this->db->fetchByAssoc($r);
```

Comment:
Can we change these variable names to something that is more meaningful?

Could we also change the $a['c']  to something that is a little more descriptive?